Makefile

```diff
@@ -199,6 +201,16 @@ dev-plugin-medialibrary:
 
 # ============================================================================
 
+# ELASTICSEARCH
+
+.PHONY: elastic
+elastic: elastic-index flush
+
+elastic-index:
+	docker-compose exec php-fpm wp elasticpress index --setup --quiet --url=www.planet4.test
```

Comment:
I think it might be better to use the `http-1` index so that
1. all tests are run on the default content (the other indexes might have more documents created by us)
2. the url may vary if we use different ports (e.g. my local index created via test site is planet4test8000-1 because I use port 8000, due to conflict with port 80). So, I still get 2 different indexes.

Reply:
Also, might be good idea to use --show-bulk-errors instead of --quiet to report any posts failing to index